src/components/pages/Users/User/User.tsx

```diff
@@ -74,8 +100,18 @@ const User = () => {
     content = <Outlet />;
   }
 
+  const generateEditUserButton = () => (
+    <Button
```

Comment:
This button is only used on one place so it doesn't need to be inside this function and can be moved inside the `controls` prop below.